### ISSUE TITLE
Warning: preg_match(): No ending delimiter '/' found in /var/www/html/ost/include/class.filter.php on line 928

### ISSUE THREAD

**Issue by Joexiaohongchen** [osTicket/osTicket#5838]
### Description

When the user tries to submit the ticket, there is a failure to submit and there is a warning message showing up. Please Help! Thank you!

![Image of Yaktocat](https://raw.githubusercontent.com/Joexiaohongchen/writing-functions/master/osticket%20issue.png)

**Comment by JediKev** [comment]
@Joexiaohongchen 

You failed to complete the full Issue Template. Next time you create an Issue please complete the _entire_ template so we have all the information required to assist you.

With this being said, looking at the error it appears one of your Ticket Filters has a regex statement that is incomplete/invalid. You will need to check all of your Filters' Rules to find the culprit. Once you find the incomplete/invalid regex statement you can then correct it and the error should go away.

Cheers.

**Comment by Joexiaohongchen** [comment]
Thank you for your reply and sorry for the incomplete issue submission. My version of osticket is 1.8. I have three custom fields in ticket details and these three fields are required, when the user wants to create a new ticket, they have to submit twice and the warning message would be displayed, like below image I attached.  I tried to check all the filters' rules but I can't find where is it. Please give me advice on how to debug it, thank you. Meanwhile, it is no error appears on the system log.

**Comment by JediKev** [comment]
@Joexiaohongchen 

Yikes..you are using a version that is wayyyy out of date and no longer supported (support is end of life). You will need to upgrade to v1.14.6 or v1.15.2 before we can help. This could be an issue that has already been addressed in the years since that release.

Cheers.
